### PR TITLE
refactor(sbb-journey-header, sbb-journey-summary): lit migration

### PR DESCRIPTION
**Pull request by TomMenga**

### CODE REVIEW

File: src/components/sbb-journey-summary/sbb-journey-summary.scss [hunk 1]
```diff
@@ -21,15 +21,14 @@
 
   margin-block-start: 0;
   margin-block-end: 0;
-  padding-inline-start: 0;
```
Comment (jeripeierSBB) [inline src/components/sbb-journey-summary/sbb-journey-summary.scss:24]: I think those changes should not appear hear. Did you maybe forgot to rebase on to lit-migration branch? (there was an update to journey summary last week)

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]
LGTM but has unwanted changes (I think)

File: src/components/sbb-journey-summary/sbb-journey-summary.scss (hunk 1 above)
Comment (TomMenga) [reply src/components/sbb-journey-summary/sbb-journey-summary.scss:24]: Not a mistake, I had to fix a spacing issue between <span> due to a difference between JSX and lit rendering

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**